services/QuillLMS/client/app/bundles/Teacher/components/cms/blog_posts/featured_blog_posts.tsx

```diff
@@ -1,42 +1,48 @@
-import React from 'react';
+import * as React from 'react';
 
-import { SortableList, } from '../../../../Shared/index'
+import { tableHeaders, blogPostRows, } from './shared'
 
-const featuredBlogPost = (blogPost, handleClickStar) => {
-  const handleClick = () => handleClickStar(blogPost.id)
-  return (
-    <div className="featured-blog-post" key={blogPost.id}>
-      <div className="featured-blog-post-header">
-        <p className="title">{blogPost.title}</p>
-        <button className="interactive-wrapper" onClick={handleClick} type="button">
-          <i className="fas fa-star" />
-        </button>
-      </div>
-      <p>{blogPost.topic}</p>
-    </div>
-  )
-}
+import { DataTable, } from '../../../../Shared/index'
 
 const FeaturedBlogPosts = ({
   featuredBlogPosts,
-  handleClickSaveOrder,
   handleClickStar,
-  updateOrder
+  updateOrder,
+  saveOrder,
 }) => {
-  const sortedBlogPosts = featuredBlogPosts.sort((bp1, bp2) => bp1.featured_order_number - bp2.featured_order_number)
-  const blogPostCards = sortedBlogPosts.map(bp => featuredBlogPost(bp, handleClickStar))
+  const [orderHasChanged, setOrderHasChanged] = React.useState(false)
+
+  function handleOrderChange(sortInfo) {
+    setOrderHasChanged(true)
+    updateOrder(sortInfo)
+  }
+
+  const rows = blogPostRows(featuredBlogPosts.sort((bp1, bp2) => bp1.featured_order_number - bp2.featured_order_number), handleClickStar, false)
+
+  let saveOrderButton = (<button className="quill-button fun contained primary focus-on-light disabled" disabled type="button">Save Order</button>)
+
+  if (orderHasChanged) {
+    saveOrderButton = (<button className="quill-button fun contained primary focus-on-light" onClick={saveOrder} type="button">Save Order</button>)
+  }
```

Comment:
This is small, but since we have this same logic in the `BlogPostTable`, we could just pull this out into a shared function in the blog_posts shared file. Something like this:

```
  function saveOrderButton({ saveOrder, orderHasChanged }) {
    const className = `quill-button fun contained primary focus-on-light ${orderHasChanged ? '' : 'disabled'}`
    return <button className={className} disabled={!orderHasChanged} onClick={saveOrder} type="button">Save Order</button>
  }
  ```